Custom project request properties (axon server context) (#47)
AxonIQ/initializr: initializr-be/src/main/resources/configuration/axon-server/saas/README.md

> ## Running in Axon SaaS

Reviewer: ```suggestion
## Running in AxonIQ Cloud
```

> * Get free access to developer context at https://cloud.axoniq.io/

Reviewer 2: ```suggestion
* Get free access to a developer context at https://cloud.axoniq.io/
```

> * Create application binding token and paste it in `src/main/resources/application.properties`

Reviewer 2: ```suggestion
* Create an application binding and paste the token in `src/main/resources/application.properties`
```
